releasenotes.txt

```diff
@@ -1,7 +1,7 @@
 Release Notes for Cppcheck 2.17
 
 New checks:
--
+- Check null pointer return from memory/resource allocation functions.
```

Comment:
Maybe the new IDs `nullPointerArithmeticOutOfMemory`, `nullPointerOutOfMemory`, `nullPointerOutOfResources` should be mentioned.

Reply:
Looks like `ctunullpointer` reports warnings related to allocation failure under the same ID.

Reply:
I can't see that `ctunullpointer` reports related warnings.

Reply:
Here's an example from daca:
~~~
flashrom-1.4.0/layout.c:199:23: error: Null pointer dereference: filename [ctunullpointer]
flashrom-1.4.0/layout.c:214:42: note: Assuming allocation function fails
flashrom-1.4.0/layout.c:214:35: note: Calling function sanitise_filename, 1st argument is null
flashrom-1.4.0/layout.c:199:23: note: Dereferencing argument filename that is null
~~~

Reply:
I have created https://trac.cppcheck.net/ticket/13521 for that issue.